imagetest/testworkflow.go

```diff
@@ -565,6 +584,7 @@ func RunTests(ctx context.Context, storageClient *storage.Client, testWorkflows
 		wg.Add(1)
 		go func(id int) {
 			defer wg.Done()
+			time.Sleep(time.Duration(id) * stagger)
```

Comment:
@dorileo 
> why not time.Sleep(stagger * time.Duration(i)) ?

I kept `id` because of this: https://go.dev/blog/loopvar-preview whole thing. `go vet` will fail it in the presubmit too.
Got rid of the unnecessary casts though.

Reply:
Ah yeah, for sure, you need to manage the go routine context access - that's fine, I meant to ask about the casts.